verify.rb

```diff
@@ -93,7 +93,7 @@ def test_img_file(img, name)
 
     # Check section alphabetization
     if websites != (sites_sort = websites.sort_by { |s| s['name'].downcase })
-      error("_data/#{section['id']}.yml not ordered by name. Correct order:" \
+      error("websites['name'] in _data/#{section['id']}.yml is not ordered alphabetically. Correct order:" \
```

Comment:
The file diff that follows this error message describes which sections need to be moved. Therefore I don’t think the website name is necessary.

Reply:
I have removed the website name per suggestion.

Reply:
Thank you! 🙂